cidr2ip/docker.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Build and upload (to Dockerhub) for all platforms for version $1.
```

Comment:
We're also using `$2` for the release, so let's add this for all of these CLI-specific docker.sh files.

```suggestion
# Build and upload (to Dockerhub) for all platforms for version $1.
# Use `-r` or `--release` after the version to also push to Dockerhub.
```